duplicate-air-sea-co2-flux-pop.ipynb

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
<p>This doesn't align with what you actual do below. It looks like you switched over to `matplotlib`.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/test_hackathon_repo/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>